launcher/src/main/scripts/bin/launcher.py

```diff
@@ -251,9 +251,15 @@ def run(process, options):
     os.execvpe(args[0], args, env)
 
 
+def need_set_inheritable():
+    # See https://docs.python.org/3/library/os.html#inheritance-of-file-descriptors
+    py_version = sys.version_info.major + .1 * sys.version_info.minor
+    return py_version >= 3.4
```

Comment:
I think this needs to be a tuple comparison. Using a number will fail when Python 3.10 comes out.